code/game/gamemodes/cult/cult.dm

```diff
@@ -191,6 +192,7 @@
 		cult_mind.current.Paralyse(5)
 		cult += cult_mind
 		cult_mind.current.cult_add_comm()
+		cult_mind.special_role = "Cultist"
```

Comment:
I wonder if we could store the "special roles" into a list instead of having this single static type. It could mean someone could be a traitor AND something else like a vampire or rev. It's unlikely/not possible now but it might be